src/pages/oauth-error-view/oauth-error-view.js

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom'
+import './oauth-error-view.scss'
+import qs from 'qs'
+import routes from "../../routes.js"
+
+const getError = search => {
+  return qs.parse(search, { ignoreQueryPrefix: true }).error_description
+}
+
+const OAuthErrorView = () => {
+  return (
+    <oauth-error-view>
+        <h2>Oops! We can't process that request. Please click <a href={routes.root}>here</a> to go back to the Operating System home page. </h2>
```

Comment:
Do we want to use the phrase "Operating System" here?  Seems Columbus-specific.

Reply:
ya ya ya